openfpga_flow/regression_test_scripts/basic_reg_test.sh

```diff
@@ -9,182 +9,191 @@ PYTHON_EXEC=python3.8
-  echo -e "Golden netlist remain unchanged"
-else
-  echo -e "Detect changes in golden scripts"; exit 1;
-fi
+run-task basic_tests/explicit_multi_verilog_files $@
 
 echo -e "Test the remove of runtime directories"
-run-task basic_tests/explicit_multi_verilog_files --debug --show_thread_logs --remove_run_dir all
-
-# Repgression test to test multi-user enviroment
-cp -r */*/basic_tests/full_testbench/configuration_chain /tmp/
-cd /tmp/ && run-task configuration_chain --debug --show_thread_logs
+clear-task-run basic_tests/explicit_multi_verilog_files $@
 
 echo -e "Testing write GSB to files";
-run-task basic_tests/write_gsb/write_gsb_to_xml --debug --show_thread_logs
-run-task basic_tests/write_gsb/write_gsb_to_xml_compress_routing --debug --show_thread_logs
-run-task basic_tests/write_gsb/write_unique_gsb_to_xml --debug --show_thread_logs
-run-task basic_tests/write_gsb/write_unique_gsb_to_xml_compress_routing --debug --show_thread_logs
+run-task basic_tests/write_gsb/write_gsb_to_xml $@
+run-task basic_tests/write_gsb/write_gsb_to_xml_compress_routing $@
+run-task basic_tests/write_gsb/write_unique_gsb_to_xml $@
+run-task basic_tests/write_gsb/write_unique_gsb_to_xml_compress_routing $@
 
 echo -e "Testing bus group features";
-run-task basic_tests/bus_group/preconfig_testbench_explicit_mapping --debug --show_thread_logs
-run-task basic_tests/bus_group/preconfig_testbench_implicit_mapping --debug --show_thread_logs
-run-task basic_tests/bus_group/full_testbench_explicit_mapping --debug --show_thread_logs
-run-task basic_tests/bus_group/full_testbench_implicit_mapping --debug --show_thread_logs
+run-task basic_tests/bus_group/preconfig_testbench_explicit_mapping $@
+run-task basic_tests/bus_group/preconfig_testbench_implicit_mapping $@
+run-task basic_tests/bus_group/full_testbench_explicit_mapping $@
+run-task basic_tests/bus_group/full_testbench_implicit_mapping $@
 
 echo -e "Testing fix pins features";
-run-task basic_tests/fix_pins --debug --show_thread_logs
+run-task basic_tests/fix_pins $@
 
 echo -e "Testing project templates";
-run-task template_tasks/vpr_blif_template --debug --show_thread_logs
-run-task template_tasks/yosys_vpr_template --debug --show_thread_logs
-run-task template_tasks/vtr_benchmarks_template --debug --show_thread_logs
+run-task template_tasks/vpr_blif_template $@
+run-task template_tasks/yosys_vpr_template $@
+run-task template_tasks/vtr_benchmarks_template $@
 
 echo -e "Testing create tsk from template and run task"
 create-task _task_copy basic_tests/generate_fabric
 run-task _task_copy
+
+echo -e "Testing output files without time stamp";
+run-task basic_tests/no_time_stamp $@
+# Run git-diff to ensure no changes on the golden netlists
+# Switch to root path in case users are running the tests in another location
+cd ${OPENFPGA_PATH}
+pwd
+git config --global --add safe.directory ${OPENFPGA_PATH}
```

Comment:
Oh ok, this makes sense now.
Can you also add `git config --global --add safe.directory *`
After this line here https://github.com/lnis-uofu/OpenFPGA/blob/140e3431a51121bf1e681af05a0ba5b961ee05b7/docker/Dockerfile.master#L24

We will have this problem solved globally

Reply:
Done